src/rpc/mempool.cpp

```diff
@@ -252,7 +255,8 @@ static RPCHelpMan testmempoolaccept()
 static std::vector<RPCResult> MempoolEntryDescription()
 {
     return {
-        RPCResult{RPCResult::Type::NUM, "vsize", "virtual transaction size as defined in BIP 141. This is different from actual serialized size for witness transactions as witness data is discounted."},
+        RPCResult{RPCResult::Type::NUM, "vsize", "maximum of sigop-adjusted size (-bytespersigop) and virtual transaction size as defined in BIP 141."},
+        RPCResult{RPCResult::Type::NUM, "vsize_bip141", "virtual transaction size as defined in BIP 141. This is different from actual serialized size for witness transactions as witness data is discounted."},
```

Comment:
I think rather than adding `vsize_bip141` to mempool RPCs, it would be better to (a) document mempool vsize as being sigop-adjusted, and (b) document rawtransaction vsize as **not** being sigop-adjusted? (Or, even better, do the sigop adjustment in cases where we can -- getrawtransaction of txs in blocks could pull up the utxo being spent via rev*.dat and calculate the sigop adjustment?)

Could be sensible to add "sigopsize" for the cases where we have a sigop count -- in that case vsize is simply max(floor((weight+3)/4), sigopsize), and the presence of a sigopsize with the exact same value as the weird vsize might be mostly self-explanatory as to what's going on. (And for the normal case where the weight dominates, it actually shows why the sigop adjustment isn't relevant)

Reply:
IIUC the suggestion is:
- Add docs for where vsize is sigop-adjusted and where it isn't.
- Add some doc/policy/?.md describing feerate, vsize, mining score
- Add an optional "sigopsize" result for `getrawtransaction` and the mempool RPCs when we're able to provide it

Will work on this